frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -776,5 +786,43 @@ export const eventUsageLogic = kea<
         reportRecordingPlayerSkipInactivityToggled: ({ skipInactivity }) => {
             posthog.capture('recording player skip inactivity toggled', { skip_inactivity: skipInactivity })
         },
+        reportNewExperimentButtonClicked: () => {
+            posthog.capture('new experiment button clicked')
+        },
+        reportExperimentCreated: ({ experiment }) => {
+            posthog.capture('experiment created', {
+                name: experiment.name,
+                id: experiment.id,
+                filters: experiment.filters,
```

Comment:
We could use the same `sanitizeFilterParams` we use for insights to reduce potential exposure of sensitive info.